pom.xml

```diff
@@ -161,6 +161,17 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.18.1</version>
+                <configuration>
+                    <forkCount>13</forkCount>
+                    <reuseForks>false</reuseForks>
+                    <testFailureIgnore>true</testFailureIgnore>
+                </configuration>
```

Comment:
In order to prevent parallel execution I would try the following settings:

```xml
<parallel>suites</parallel>
<forkCount>1</forkCount>
<threadCount>1</threadCount>
```